src/components/CustomCollection/EditableInput.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react'
+
+type PropTypes = {
+  text: string
+  placeholder: string
+  children: React.ReactNode
+  childRef: React.RefObject<HTMLInputElement>
+  onSave: (s: string) => void
+}
+export const EditableInput = ({
```

Comment:
I have naming ideas... I would say _either_ we commit to this component being abstracted for the purpose of editing text inline, and call it `InlineTextInput` (and move it outside of `CustomCollection) _or_ we commit to this component specifically being used for the Collection title (which can be edited inline) and call it something to the effect of `EditableCollectionTitle`